src/utils/styleConstants.ts

```diff
@@ -6,11 +6,12 @@ export const FONT_WEIGHTS = {
 };
 
 export const FONT_SIZES = {
-  H1: "32px",
```

Comment:
According to the typography, it is fine for the H1 to be 32, however, there is another much larger title that is 70px. It would be good if you spoke to the people who made the figma so that the typography remains in order and you do not have many problems.

![image](https://github.com/user-attachments/assets/9b9542b1-e6d2-461b-aad1-6ee196ffafd2)

Reply:
In this case as h1 is the larger text element, I changed it to 64px, which was the last update in the figma if you check, 32px is in red line because it was changed.

Reply:
Great, if it could be changed